darc/didresolver.go

```diff
@@ -0,0 +1,172 @@
+package darc
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"github.com/mr-tron/base58"
```

Comment:
please use the same base58 already used by aries-framework-go, so as to reduce the dependencies: github.com/btcsuite/btcutil/base58

See also: https://github.com/hyperledger/aries-framework-go/issues/1192